deploy-manage/deploy/elastic-cloud/tools-apis.md

```diff
@@ -20,6 +20,37 @@ mapped_urls:
 
 ⚠️ **This page is a work in progress.** ⚠️
 
-The documentation team is working to combine content pulled from the following pages:
+% The documentation team is working to combine content pulled from the following pages:
 
-* [/raw-migrated-files/docs-content/serverless/elasticsearch-http-apis.md](/raw-migrated-files/docs-content/serverless/elasticsearch-http-apis.md)
\ No newline at end of file
+% * [/raw-migrated-files/docs-content/serverless/elasticsearch-http-apis.md](/raw-migrated-files/docs-content/serverless/elasticsearch-http-apis.md)
+
+# REST APIs [elasticsearch-http-apis]
+
+
+## API references [elasticsearch-api-references-links] 
+
+The following APIs are available for {{es-serverless}} users. These links will take you to the autogenerated API reference documentation.
+
+[{{es-serverless}} APIs →](https://www.elastic.co/docs/api/doc/elasticsearch-serverless)
+:   Use these APIs to index, manage, search, and analyze your data in {{es-serverless}}.
+
+    ::::{tip} 
+    Learn how to [connect to your {{es-serverless}} endpoint](../../../solutions/search/get-started.md).
+
+    ::::
+
+
+[{{kib}} Serverless APIs →](https://www.elastic.co/docs/api/doc/serverless)
+:   Use these APIs to manage resources such as connectors, data views, and saved objects for your {{serverless-full}} project.
+
+[{{serverless-full}} APIs →](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless)
+:   Use these APIs to manage your {{serverless-full}} projects.
+
+
+## Additional API information [additional-api-details] 
+
+[{{es}} API conventions](https://www.elastic.co/guide/en/serverless/current/elasticsearch-api-conventions.html)
+:   Reference information about headers and request body conventions for {{es-serverless}} REST APIs.
+
+[Management API conventions](https://www.elastic.co/guide/en/serverless/current/elasticsearch-kibana-api-conventions.html)
```

Comment:
should try to change these to relevant asciidocalypse or api ref links

Reply:
The Elasticsearch API conventions are here: https://docs-v3-preview.elastic.dev/elastic/asciidocalypse/tree/main/elasticsearch/docs/reference/elasticsearch/rest-apis/api-conventions

The Kibana page is no longer necessary, however, since it matches what's in the intro and authentication pages in the API reference site. So IMO that Kibana link should be removed.